code/modules/mob/living/carbon/superior_animal/attack.dm

```diff
@@ -62,15 +62,19 @@
 	if(rapid)
 		spawn(6)
 			Shoot(target, loc, src)
+			sleep(1) //Lets make sure we shoot so we add a small stop gap to not go to fast
 			handle_ammo_check()
 		spawn(10)
 			Shoot(target, loc, src)
+			sleep(1)
 			handle_ammo_check()
 		spawn(14)
 			Shoot(target, loc, src)
+			sleep(1)
 			handle_ammo_check()
 	else
 		Shoot(target, loc, src)
```

Comment:
We had big issues with putting sleep in mobs just recently, this seems like a very bad idea.

Reply:
No its not, this are micro in limited mobs that will be in limited uses. This fixes *more* then it breaks.
adding a second spawn their wouldnt have the same affect
The reason why sleep was bad before is that 200+ mobs were using it all at the same time, all they time. Well this is more limited